client/tests/sdk/test_model.py

```diff
@@ -7,18 +7,17 @@
 
 from pyfakefs.fake_filesystem_unittest import TestCase
 
+from starwhale import get_data_loader, PipelineHandler, UserRawDataLoader
 from starwhale.consts import DEFAULT_PROJECT
 from starwhale.base.uri import URI
 from starwhale.utils.fs import ensure_dir
-from starwhale.api.model import PipelineHandler
 from starwhale.base.type import (
     URIType,
     RunSubDirType,
     DataFormatType,
     DataOriginType,
     ObjectStoreType,
 )
-from starwhale.api.dataset import get_data_loader, UserRawDataLoader
 from starwhale.api._impl.job import Context
```

Comment:
move this to 'from starwhale import Context,get_data_loader, PipelineHandler, UserRawDataLoader'?

Reply:
done